google-cloud-vision/lib/google/cloud/vision/v1/image_annotator_client.rb

```diff
@@ -136,7 +136,7 @@ def initialize \
               updater_proc = credentials.updater_proc
             end
 
-            package_version = Gem.loaded_specs['google-cloud-vision'].version.version
+            package_version = Gem.loaded_specs['google-cloud'].version.version
```

Comment:
Revert

Reply:
@dazuma Yeah, basically just wanted an easy way to share what's going on.